src/commands/Find.thy

```diff
@@ -0,0 +1,75 @@
+theory Find
+  imports Main
+begin
+
+section \<open>Find theorems with @{command find_theorems}\<close>
+text \<open>
+  Theorems can be searched by name where @{verbatim *} wildcards are allowed.
```

Comment:
There is also the shorter form of this antiquotation, which makes the source more readable.
```suggestion
  Theorems can be searched by name where ▩‹*› wildcards are allowed.
```